pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -818,20 +818,10 @@ func (b *MasqueradeBindMechanism) preparePodNetworkInterfaces(queueNumber uint32
 		return err
 	}
 
-	if Handler.HasNatIptables(iptables.ProtocolIPv4) || Handler.NftablesLoad("ipv4-nat") == nil {
-		err = Handler.ConfigureIpv4Forwarding()
-		if err != nil {
-			log.Log.Reason(err).Errorf("failed to configure ipv4 forwarding")
-			return err
-		}
-
-		err = b.createNatRules(iptables.ProtocolIPv4)
-		if err != nil {
-			log.Log.Reason(err).Errorf("failed to create ipv4 nat rules for vm error: %v", err)
-			return err
-		}
-	} else {
-		return fmt.Errorf("Couldn't configure ipv4 nat rules")
+	err = b.createNatRules(iptables.ProtocolIPv4)
+	if err != nil {
+		log.Log.Reason(err).Errorf("failed to create ipv4 nat rules for vm error: %v", err)
+		return err
```

Comment:
We don't care about ipv6 only clusters right ?

Reply:
right